databases/postgres.go

```diff
@@ -0,0 +1,64 @@
+package databases
+
+import (
+	"database/sql"
+	_ "github.com/lib/pq"
+	"github.com/zzuun/time-tracker/models"
+	"time"
+)
+
+type Postgres struct {
```

Comment:
Pls don't name after specific technology. 
Rename package to: datastore or db
the interface to Store or Datastore
and this struct to database